README.md

```diff
@@ -365,4 +365,4 @@ See https://github.com/jenkinsci/gitlab-plugin/tree/master/src/docker/README.md
 
 # Release Workflow
 
-To perform a full plugin release, maintainers can run ``mvn release:prepare release:perform`` To release a snapshot, e.g. with a bug fix for users to test, just run ``mvn deploy``
```

Comment:
I don't know what GitHub's online editor is doing here... I think it's chopping empty lines at the end of the file?